gemet/thesaurus/migrations/0004_version.py

```diff
@@ -14,6 +16,13 @@ def forwards_func(apps, schema_editor):
     try:
         latest_concept = Concept.objects.using(db_alias).latest('date_changed')
     except Concept.DoesNotExist:
+        # version created for import
+        current_date = timezone.now().strftime("%Y-%m-%d %H:%M:%S%z")
+        Version.objects.create(
```

Comment:
Create only the `publication_date` under `except`. Creating the version means duplicate code.